src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -562,6 +581,9 @@ type SchemaRegistryAPI struct {
 	External *SchemaRegistryExternalConnectivityConfig `json:"external,omitempty"`
 	// TLS is the configuration for schema registry
 	TLS *SchemaRegistryAPITLS `json:"tls,omitempty"`
+	// AuthenticationMethod can enable authentication method per schema registry
+	// listener. Available options are: none, http_basic.
+	AuthenticationMethod string `json:"authenticationMethod,omitempty"`
```

Comment:
for schema registry it enables it on all listeners, right?

Reply:
We have only one listener.